packages/tools/pie-icons-vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@justeattakeaway/pie-icons-vue",
-  "version": "4.5.0",
```

Comment:
@ashleynolan hey, I think if we don't add other icon packages to the changeset, only vue/react/config gets updated and this is in line with [changeset docs](https://github.com/changesets/changesets/blob/main/docs/linked-packages.md). icon-react and icons-vue update automatically only when their dependency pie-icons updates. But because packages are linked after this version bump for icons-vue package, if we introduce some changes to icons-react package, it will be bumped to 4.7.0 after 4.5.0 and again, it is in line with the changeset docs.

Reply:
does that mean we should not merge this until that's addressed?

Reply:
spoke to @siggerzz about this and we're good to merge 😄 the bump to pie-icons-vue is expected and there is no bump to the react icons package here so this does not need to be blocked for now. If the next react-icon change skips 4.6.0 then this definitely needs investigating but it is not a current blocker for these releases 👍